src/ray/common/ray_config_def.h

```diff
@@ -329,3 +329,7 @@ RAY_CONFIG(bool, put_small_object_in_memory_store, false)
 /// the owner has been granted a lease. A value >1 is used when we want to enable
 /// pipelining task submission.
 RAY_CONFIG(uint32_t, max_tasks_in_flight_per_worker, 1)
+
+RAY_CONFIG(bool, run_profiling,
+           getenv("RAY_PROFILE_ENABLED") != nullptr &&
+               getenv("RAY_PROFILE_ENABLED") == std::string("true"))
```

Comment:
We probably shouldn't feature flag it via environment variable? Seems to me just enable it via _internal_config is better.

Reply:
One concern I have is that `_internal_config` (not sure if it is still the case) is not meant to be exposed to users, but this feature is supposed to be exposed to users.